apihit.py

```diff
@@ -0,0 +1,37 @@
+import os
+import requests
+import json
+from .models import (DBSession, Entry, NewEntry, LoginPage)
+
+
+def get_journal_entries():
+    url = "https://sea401d2.crisewing.com/api/export?apikey="
+    key = os.environ.get("JOURNAL_KEY", "")
+    params = {"username": "SeleniumK"}
```

Comment:
not required by the api.  But that's okay.  It won't kill the call to have the data present.